test/functional-portable/corerp/noncloud/resources/recipe_terraform_test.go

```diff
@@ -143,7 +143,7 @@ func Test_TerraformRecipe_KubernetesPostgres(t *testing.T) {
 	template := "testdata/corerp-resources-terraform-postgres.bicep"
 	appName := "corerp-resources-terraform-pg-app"
 	envName := "corerp-resources-terraform-pg-env"
-	extenderName := "corerp-resources-terraform-pg"
+	extenderName := "pgs-resources-terraform-pgsapp"
```

Comment:
Let's keep prefix the same for consistency with other resource names for now unless just changing the suffix doesn't help :)

```suggestion
	extenderName := "corerp-resources-terraform-pgsapp"
```

Reply:
changed :)